README.md

```diff
@@ -5,18 +5,17 @@
 [![PyPi](https://img.shields.io/pypi/pyversions/meeshkan)](https://pypi.org/project/meeshkan/)
 [![License](https://img.shields.io/pypi/l/meeshkan)](LICENSE)
 
-Meeshkan is a tool for mocking HTTP APIs for use in sandboxes as well as for automated and exploratory testing. It uses a combination of API definitions, recorded traffic and code in order to make crafting mocks as enjoyable as possible.
+Meeshkan is a tool that mocks HTTP APIs for use in sandboxes as well as for automated and exploratory testing. It uses a combination of API definitions, recorded traffic and code in order to make crafting mocks as enjoyable as possible.
 
 Meeshkan requires Python 3.6 or later.
```

Comment:
```suggestion
```

This doesn't need to be here as it's already mentioned in the `Installation` step.

What would be nice here, is something about _who_ should be using Meeshkan or the best use cases. So some shorter version of the `Who should use Meeshkan?` section of the announcement could work.